ui/app/routes/vault/cluster/secrets/backend/list.js

```diff
@@ -80,7 +80,7 @@ export default Route.extend({
       ssh: 'role-ssh',
       transform: this.modelTypeForTransform(tab),
       aws: 'role-aws',
-      pki: tab === 'certs' ? 'pki/pki-certificate' : 'pki/pki-role',
+      pki: `pki/${tab || 'pki-role'}`,
```

Comment:
this ultimately is why I have this PR. It gets me setup for a much easier time to add new queryParams that are associated by tab name and all the files can be kept in the pki folder. I have three additional tabs I need to add in the PKI project. I am following the keymgmt pattern, which I really like.